<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBooking</class>
 <widget class="QDialog" name="AddBooking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>450</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Booking Type</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rentalCarReservationButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="addToNewTravel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Add to a new travel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hotelBookingButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="flightBookingButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="bookingAttributePages">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>431</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="rentalCarReservierungPage">
    <widget class="QDoubleSpinBox" name="PriceCarEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>96</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>9999999999.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>40</y>
       <width>47</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>End Date</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>68</y>
       <width>26</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="startDateCarEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>79</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>12</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Date</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="endDateCarEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>79</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="returnLocEdit">
     <property name="geometry">
      <rect>
       <x>92</x>
       <y>145</y>
       <width>133</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>145</y>
       <width>84</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Return Location</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="companyEdit">
     <property name="geometry">
      <rect>
       <x>92</x>
       <y>181</y>
       <width>133</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="pickupLocEdit">
     <property name="geometry">
      <rect>
       <x>92</x>
       <y>109</y>
       <width>133</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>109</y>
       <width>85</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pickup Location</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>217</y>
       <width>67</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Vehicle Class</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>181</y>
       <width>52</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Company</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="vehicleClassEdit">
     <property name="geometry">
      <rect>
       <x>92</x>
       <y>217</y>
       <width>133</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="pred1CarEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>250</y>
       <width>133</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>74</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Predecessor 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>280</y>
       <width>74</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Predecessor 2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pred2CarEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>280</y>
       <width>133</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="flightBookingPage">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Date</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="startDateFlightEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>79</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>47</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>End Date</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="endDateFlightEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>79</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="priceFlightSpinbox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>96</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>9999999999.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>26</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="toCodeEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>133</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="bookingClassEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>200</y>
       <width>133</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="fromCodeEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>133</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="airlineEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>133</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>201</y>
       <width>74</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Booking Class</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>105</y>
       <width>34</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Airline</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>169</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>To (Airport Code)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>137</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>From (Airport Code)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pred1FlightEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>230</y>
       <width>133</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="pred2FlightEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>260</y>
       <width>133</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>230</y>
       <width>74</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Predecessor 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>260</y>
       <width>74</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Predecessor 2</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="hotelBookingPage">
    <widget class="QDoubleSpinBox" name="priceHotelEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>96</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>9999999999.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>30</y>
       <width>47</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>End Date</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>58</y>
       <width>26</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>2</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Date</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="startDateHotelEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>79</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="endDateHotelEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>79</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>27</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Stadt</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Hotelname</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="hotelEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>311</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="roomTypeEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>311</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="townEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>311</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Zimmertyp</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pred1HotelEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>133</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>74</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Predecessor 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>74</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Predecessor 2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pred2HotelEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>210</y>
       <width>133</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QCheckBox" name="addToSelectedTravel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Add to current selected travel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddBooking</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddBooking</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
